src/gobstuf/rest/brp/views.py

```diff
@@ -15,13 +15,18 @@ class IngeschrevenpersonenView:
         'partners'
     ]
 
+    @property
+    def functional_query_parameters(self):
+        return {
+            **super().functional_query_parameters,
+            'inclusiefoverledenpersonen': False,
+        }
+
 
 class IngeschrevenpersonenFilterView(IngeschrevenpersonenView, StufRestFilterView):
     request_template = IngeschrevenpersonenFilterStufRequest
     response_template = IngeschrevenpersonenStufResponse
-    response_template_properties = {
-        'inclusiefoverledenpersonen': False,
-    }
```

Comment:
I think this is the only place this was used; the usage of response_template_properties can be removed from the code too.

Reply:
Removed